Rivet/Functions/Get-RivetConfig.ps1

```diff
@@ -40,28 +40,49 @@ function Get-RivetConfig
     )
 
     Set-StrictMode -Version 'Latest'
+    Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
 
     function Resolve-RivetConfigPath
     {
         [CmdletBinding()]
         param(
-            [Parameter(Mandatory=$true,ValueFromPipeline=$true)]
+            [Parameter(Mandatory,ValueFromPipeline)]
             [string]
             # The path from the rivet config file to resolve.
-            $Path
+            $ConfigPath,
+
+            [Switch]
+            # The path *must* exist, so resolve it.
+            $Resolve

```

Comment:
The purpose of this function is to convert a path relative to the `rivet.json` file into a full path. There are two paths (at the moment) in Rivet configuration: the path to the migrations directory and the path to any plug-ins. The migrations directory doesn't need to exist. Rivet creates it for you if it is missing. The plug-ins directory *does* need to exist. In order to support wildcard-ed version pinning to these plug-ins (e.g. `PSModules\MyPlugins\*\MyPlugins.psd1`), the plug-in path needs to support wildcards.

Are you suggesting replacing the `Resolve` parameter with a `ConfigLiteralPath` parameter and resolving the migrations path would use `ConfigLiteralPath` and resolving the plug-in path would use `ConfigPath`?

Reply:
Yes, that was my initial recommendation but after you elaborated a bit more in regards to when/why you would use the `-Resolve` switch, it does make more sense to keep it as-is rather than change to `-ConfigLiteralPath` and `-ConfigPath`.